src/crimson/os/seastore/extent_placement_manager.cc

```diff
@@ -1002,54 +1002,77 @@ RandomBlockOolWriter::do_write(
          t, extents.size());
   std::vector<write_info_t> writes;
   writes.reserve(extents.size());
+
   for (auto& ex : extents) {
-    auto paddr = ex->get_paddr();
-    assert(paddr.is_absolute());
-    RandomBlockManager * rbm = rb_cleaner->get_rbm(paddr); 
-    assert(rbm);
-    TRACE("write extent {}, paddr {} ...",
-          fmt::ptr(ex.get()), paddr);
     auto& stats = t.get_ool_write_stats();
     stats.extents.num += 1;
     stats.extents.bytes += ex->get_length();
-    stats.num_records += 1;
-
     ex->prepare_write();
-    extent_len_t offset = 0;
+
+    RandomBlockManager * rbm = rb_cleaner->get_rbm(ex->get_paddr()); 
+    assert(rbm);
+    paddr_t cur_base = ex->get_paddr();
+    assert(ex_base.is_absolute());
+    extent_len_t cur_len = ex->get_length();
     bufferptr bp;
+    TRACE("write extent {}, paddr {} ...",
+	  fmt::ptr(ex.get()), cur_base);
```

Comment:
Seems the logic above is unchanged.

Drop unnecessary/unrelated changes to save review efforts, unless the change makes it more readable.

Reply:
Done.